packages/backend/src/tokens/request.ts

```diff
@@ -195,16 +207,36 @@ ${error.getFullMessage()}`,
   }
 
   async function refreshToken(authenticateContext: AuthenticateContext): Promise<string> {
-    // To perform a token refresh, apiClient must be defined.
-    assertApiClient(options.apiClient);
+    try {
+      // To perform a token refresh, apiClient must be defined.
+      assertApiClient(options.apiClient);
+    } catch (err: any) {
+      throw { errors: [{ code: RefreshTokenErrorReason.MissingApiClient, message: err?.message || '' }] };
```

Comment:
Why the mix between single `error` and `errors` array? I've only seen single errors, not multiple, in your code below 🤔

Reply:
@LekoArts an `errors` array is returned on API errors.

Reply:
Just refactored this, to be clearer.